### ISSUE TITLE
Add brave://settings UX to allow user to re-enable site communication with local services

### ISSUE THREAD

**Issue by pes10k** [brave/brave-browser#11733]
As of https://github.com/brave/adblock-lists/pull/463 brave now blocks requests from websites to local resources. We should add the ability in brave://settings for advanced users to re-enable this uncommon use case, and allow websites to communicate with local processes.

On Android, this will be blocked by https://github.com/brave/brave-browser/issues/9536

This will also be blocked by a TBD adblock-rust issue to add tags to the relevant filter rules

**Comment by pes10k** [comment]
closing in favor of https://github.com/brave/brave-browser/issues/27346
